frontend/dashboard/pages/CreateService/CreateService.test.tsx

```diff
@@ -85,15 +83,16 @@ describe('CreateService', () => {
 
   it('should show error message that app name is invalid when it contains invalid characters', async () => {
     const user = userEvent.setup();
-    render();
+    const { container } = render();
 
     await waitForElementToBeRemoved(() => screen.getByText('dashboard.loading'));
 
+    await act(() => user.click(screen.getByRole('combobox')));
     await act(() => user.click(screen.getByRole('option', { name: /user_full_name/i })));
-    await act(() => user.type(screen.getByRole('textbox'), 'datamodels'));
+    await act(() => user.type(container.querySelector('#service-saved-name'), 'datamodels'));
 
-    const createBtn = await screen.findByText('dashboard.create_service_btn');
-    await act(() => user.click(createBtn));
+    const createButton = await screen.findByText('dashboard.create_service_btn');
```

Comment:
Ser at du ikke har gjort store endringer her, men kunne vi gjort sånn som dette? :)
```suggestion
    const createButton = await screen.queryByRole('button', { name: 'dashboard.create_service_btn' });
```